app/controllers/admin/npq/applications/eligible_for_funding_controller.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+module Admin
+  module NPQ
+    module Applications
+      class EligibleForFundingController < Admin::BaseController
+        StatusOption = Struct.new(:value, :label)
+
+        skip_after_action :verify_authorized
+        skip_after_action :verify_policy_scoped
+
+        def edit
+          @status_options = [
+            StatusOption.new(true, "Yes"),
+            StatusOption.new(false, "No"),
+          ]
+          @npq_application = NPQApplication.find(params[:id])
+        end
+
+        def update
+          npq_application = NPQApplication.find(params[:id])
+          npq_application.assign_attributes(eligible_for_funding_params)
+
+          case eligible_for_funding_params["eligible_for_funding"]
+          when "true"
+            npq_application.funding_eligiblity_status_code = :marked_funded_by_policy
+          when "false"
+            npq_application.funding_eligiblity_status_code = :marked_ineligible_by_policy
+          end
```

Comment:
These are options already in the DB and show that they have had manual intervention.

Reply:
What I mean is why aren't we using these in the form then, rather than true/false and then needing this translation layer. If the form just used marked_funded_by_policy and marked_ineligible_by_policy as the values on the radios then it could be fed directly through.